validator/db/kv/attestation_history_new.go

```diff
@@ -82,7 +83,7 @@ func (hd EncHistoryData) getTargetData(ctx context.Context, target uint64) (*His
 
 	history.Source = bytesutil.FromBytes8(hd[cursor : cursor+sourceSize])
 	sr := make([]byte, 32)
-	copy(hd[cursor+sourceSize:cursor+historySize], sr)
+	copy(sr, hd[cursor+sourceSize:cursor+historySize])
```

Comment:
why is this inverted ?

Reply:
first var is destination, it was a bug. discovered by import data test

Reply:
added this to the get target data test